client/src/app/shared/components/legal-notice-content/legal-notice-content.component.html

```diff
@@ -1,6 +1,9 @@
 <mat-card class="os-card">
     <div>
-        <div class="legal-notice-text" [innerHtml]="legalNotice"></div>
+        <div *ngIf="legalNotice" class="legal-notice-text" [innerHtml]="legalNotice"></div>
+        <div *ngIf="!legalNotice" translate>
```

Comment:
The `component.ts` has `this.translate.instant(...)` for the legal notice. Don't double-translate.